### ISSUE TITLE
No of epochs to be trained for achieving the given results

### ISSUE THREAD

**Issue by pratikm141** [cyvius96/prototypical-network-pytorch#3]
Hi, I was able to reach 48.2% accuracy for 1-shot on the test set after 200 epochs. Can you please mention the number of epochs that you trained the model to achieve the given 1-shot and 5-shot results. Was there any other architectural changes made for getting these results.

**Comment by yinboc** [comment]
The result was got by runing `python train.py` and `python test.py`, that is, the default setting (200 epochs).

The test script would have some variance (it samples 2000 batches), but ideally won't be very large. 
Does a re-trained model get the similar result (~48%)?

**Comment by pratikm141** [comment]
Thanks for the response. I got the reported accuracy on retraining the model.
